Source/FullScreenPokemon.ts

```diff
@@ -2039,8 +2039,8 @@ module FullScreenPokemon {
             if (!thing.cycles || !(<any>thing.cycles).walking) {
                 thing.FSP.TimeHandler.addClassCycle(
                     thing,
-                    ["walking", "standing"],
-                    "walking",
+                    !(<IPlayer>thing).cycling ? ["walking", "normal"] : ["cycling", "normal"],
+                    !(<IPlayer>thing).cycling ? "walking" : "cycling",
```

Comment:
There's a better way of doing this.

In `startCycling`, add a `"cycling"` class to the player, and remove it in `stopCycling`.